### PR TITLE
[disaster dashboard] ranking section

### PR DESCRIPTION
**Pull request by chejennifer**
code for the ranking component

This is one part of the changes for the disaster dashboard MVP here on dev:
https://dev.datacommons.org/dev/disaster-dashboard

### CODE REVIEW

File: static/js/disaster_dashboard/ranking_section.tsx [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+              </div>
+            );
+          })}
+        </>
+      )}
+    </div>
+  );
+}
+
+function getRankedDisasterCounts(
```
Comment (shifucun) [inline static/js/disaster_dashboard/ranking_section.tsx:108]: Add comments for high level logic for these functions?

Comment (shifucun) [reply static/js/disaster_dashboard/ranking_section.tsx:108]: If time allowed, would be nice to add simple test for these helper functions, so new people can update the code with more confidence.

Comment (chejennifer) [reply static/js/disaster_dashboard/ranking_section.tsx]: done

File: static/js/disaster_dashboard/ranking_section.tsx [hunk 2]
```diff
@@ -0,0 +1,182 @@
...
+          })}
+        </>
+      )}
+      {(!_.isEmpty(rankingUnits) ||
+        props.selectedDisaster !== DisasterType.ALL) && (
+        <>
+          <h3>Rankings</h3>
+          {rankingUnits.map((rankingUnit) => {
+            return (
+              <div
```
Comment (beets) [inline static/js/disaster_dashboard/ranking_section.tsx:85]: We should make this regular `<a>` links and work on a url structure for navigating the dashboard for the next iteration.

Comment (chejennifer) [reply static/js/disaster_dashboard/ranking_section.tsx:85]: added TODO in https://github.com/datacommonsorg/website/pull/1832

**Pull request merged**